go/libkb/sig_chain.go

```diff
@@ -411,7 +422,16 @@ func (sc *SigChain) verifySubchain(ctx context.Context, kf KeyFamily, links []*C
 			continue
 		}
 
-		newKID := link.GetKID()
+		if link.IsStubbed() {
+			if first {
+				return cached, cki, SigchainV2StubbedFirstLinkError{}
+			}
+			if link.NeedsSignature() {
```

Comment:
Would it make sense to unify the `isModifyingKeys || isFinalLink || hasRevocations` checks below with this `NeedsSignature` interface?